examples/react/w3console/src/components/SpaceRegistrar.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+import { useKeyring } from '@w3ui/react-keyring';
+import Loader from './Loader';
```

Comment:
We should have a linter enforce the use of semicolons or not, right now things are a mix.

Reply:
great call - filed an issue here: https://github.com/web3-storage/w3ui/issues/510